Source/Charts/Charts/ChartViewBase.swift

```diff
@@ -529,7 +529,15 @@ open class ChartViewBase: NSUIView, ChartDataProvider, AnimatorDelegate
             }
             else
             {
-                _indicesToHighlight = [h!]
+                if (_indicesToHighlight == [h!])
```

Comment:
I think you will need a switch to be able control the  behavior. not every one needs this I suppose?